File: DS-Algo-with-Java/Lecture-12 Arrays/FindPeakInArray.java
public class FindPeakInArray {
    public static void main(String[] args) {
        int arr[] = {1, 1, 3, 4, 2, 3, 5, 7, 0};
        int result = 0;

        for (int i = 1; i < arr.length - 1; i++) {
            if (arr[i - 1] < arr[i] && arr[i + 1] < arr[i]) {
                if (arr[i] > result) {
                    result = arr[i];
                }
            }
        }

        System.out.println(result);
    }
}

File: DS-Algo-with-Python/Lecture-12 Arrays/FindPeakInArray.py
from array import *

arr = array('i', [12, 1, 33, 4, 9, 3, 5, 7, 120])
result = 0

for i in range(1, len(arr) - 1):
    if arr[i - 1] < arr[i] and arr[i + 1] < arr[i]:
        if arr[i] > result:
            result = arr[i]

print(result)
